src/unity/python/turicreate/toolkits/evaluation.py

```diff
@@ -33,6 +33,19 @@
                                               _check_categorical_option_type
 from turicreate.toolkits._main import ToolkitError as _ToolkitError
 
+try:
+    from UserDict import UserDict
+except ImportError:
+    from collections import UserDict
+
+class _Evaluation(dict):
```

Comment:
The return value from image_classifier `evaluate` should probably inherit from this type, right? (The whole point of this PR is to fix API inconsistency, meaning all evaluate methods should return the same base type -- `isinstance` should behave the same on all of them.)

Reply:
I changed it. As for what the behavior should be. It doesn't currently break anything I know of, it just wraps the dictionary object in python. That way the user still gets an explore warning in TuriCreate when they try to explore their evaluations.